mesheryctl/internal/cli/root/system/check.go

```diff
@@ -581,6 +581,20 @@ func mesheryReadinessHealthCheck() (bool, error) {
 	return true, nil
 }
 
+// mesheryNsDeleteCheck waits for Meshery namespace to be deleted, returns (done, error)
+func mesheryNsDeleteCheck() (bool, error) {
+	kubeClient, err := meshkitkube.New([]byte(""))
+	if err != nil {
+		return false, err
+	}
+
+	if err := utils.WaitForNamespaceDeleted(kubeClient, utils.MesheryNamespace, 300); err != nil {
+		return false, err
+	}
+
+	return true, nil
+}
+
```

Comment:
shift this to healthcheck.go in utils, this function doesn't align with other functions in check.go

Reply:
Got it